developer/contributing.rst

```diff
@@ -100,8 +100,12 @@ Moreover, keep in mind the following guidelines:
 - after pushing your branch code, make a pull-request of that
   corresponding change of yours which should contain a descriptive
   message and mention the issue number as suggested in the example above
-- try to keep commits to a minimum. If you work on one issue squash any
-  commits you have made into one informative commit.
+- try to keep single feature/bug-fix per PR and each PR should 
```

Comment:
I would use "make sure" instead of "try", because most likely if anyone works on multiple features in a single PR we'll reject it. One feature, one PR. A big feature may need other minor related features, but it's still one logical change.

Different features should never go in one PR.

Make sure to send one pull request for each feature. Each pull request should contain one commit.